response_operations_ui/exceptions/error_codes.py

```diff
@@ -0,0 +1,26 @@
+from enum import Enum
+
+
+class ErrorCode(Enum):
+    """
+    E0001 to E0006 relate to API Error Codes
+    """
+
+    API_CONNECTION_ERROR = "E0001"
```

Comment:
I'm not a huge fan of this pattern, I think it risks ending up as a random artifact that doesn't end up proliferated throughout the service. I also think having to call `get_error_code_message` to get the actual useful bit of the error is onerous.